src/test/java/com/app/library/MapperTest.java

```diff
@@ -0,0 +1,197 @@
+package com.app.library;
+
+import com.app.library.model.dto.AuthorDto;
+import com.app.library.model.dto.BookCategoryDto;
+import com.app.library.model.dto.UserDto;
+import com.app.library.model.entity.AuthorEntity;
+import com.app.library.model.entity.BookCategoryEntity;
+import com.app.library.model.entity.BookEntity;
+import com.app.library.model.entity.UserEntity;
+import com.app.library.model.mapper.AuthorMapper;
+import com.app.library.model.mapper.BookCategoryMapper;
+import com.app.library.model.mapper.UserMapper;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.List;
+
+@SpringBootTest
```

Comment:
I think annotation `@SpringBootTest` is redundant here

Reply:
Done